spec/Collection/ProcessArgumentsCollectionSpec.php

```diff
@@ -122,4 +122,16 @@ function it_should_be_able_to_add_an_argument_with_comma_separated_files()
 
         $this->getValues()->shouldBe(['--argument=file1.txt,file2.txt']);
     }
+
+    function it_should_be_able_to_add_boolean_nullable_argument()
```

Comment:
I would make three separate tests. Now you need to take the previous result into account.

Reply:
@Landerstraeten sure! Don't go anywhere, I'll fix it right now :)